src/ui/Pricing/SubscriptionSettings/SubscriptionCard/Card.svelte

```diff
@@ -147,15 +147,59 @@
     display: inline-flex;
     --bg: var(--primary, var(--green));
     --bg-hover: var(--primary-hover, var(--green-hover));
+    --v-padding: 8px;
   }
 
   .btn-2 {
     --bg: var(--white);
+    --v-padding: 7px;
   }
 
   .billing {
     position: absolute;
     bottom: 16px;
     right: 16px;
   }
+
+  :global(.phone-xs),
+  :global(.phone) {
+    h2 {
+      font-weight: 400;
+    }
+    .actions {
+      --margin: 16px;
+      flex-direction: column;
+      align-items: flex-start;
+    }
+
+    .btn-mobile {
+      font: var(--body-2);
+    }
+
+    .subaction {
+      margin: 8px 0 0 0;
+    }
+
+    h4.caption {
+      transform: translateY(2px);
+    }
+
+    .caption {
+      font-size: 14px;
+      line-height: 20px;
+    }
+
+    .billing {
+      position: static;
+      display: flex;
+      align-items: center;
+      margin-top: 12px;
+      font-size: 14px;
+      line-height: 20px;
```

Comment:
Repetition of similar issue that was discussed here https://github.com/santiment/san-webkit/pull/93#discussion_r1084897861

Reply:
We don't have variable for 14px/20px